### PR TITLE
Disco Pane content update - Feb. 2017

### LINKED ISSUE #4546: Disco Pane content update - Feb. 2017
For the **16 February** push, can we please REMOVE:

- OmniSidebar
- all three current themes

... and REPLACE them with Video DownloadHelper and three new themes.

**Download videos** 
with [Video DownloadHelper by mig](https://addons.mozilla.org/firefox/addon/video-downloadhelper/)
Easily grab videos from YouTube, Dailymotion, Facebook, and most other video sites with this simple download tool.

[themes]

Books [by Josterix](https://addons.mozilla.org/firefox/addon/color-books/)
Yellowfield [by g650609](https://addons.mozilla.org/firefox/addon/yellowfield-1/)
Fon615 [by levscaya16](https://addons.mozilla.org/firefox/addon/fon615-by-levscaya/)

Also, could we please change up the ordering of content? Full lineup to look like so:

Books [theme]
Video DownloadHelper
uBlock
Yellow field 1 [theme]
Enhance YouTube
Emoji Keyboard
Fon615 [theme]

### PR DESCRIPTION
**Pull request by diox**
Fix #4546

### CODE REVIEW

**Comment by EnTeQuAk** [comment]
Needs the `Fix #4546` treatment

File: src/olympia/discovery/data.py [hunk 1]
```diff
@@ -11,47 +11,39 @@ def __init__(self, *args, **kwargs):
...
-        addon_id=296534,
-        heading=_(u'Easily access bookmarks {start_sub_heading}with '
-                  u'{addon_name}{end_sub_heading}'),
-        description=string_concat(
-            '<blockquote>',
-            _(u'Are you constantly scrolling through your bookmarks? '
-              u'Bring your lists into view with a single, simple gesture.'),
-            '</blockquote>')),
+    # 'yellowfield 1' theme.
+    DiscoItem(addon_id=599252, addon_name='Yellow Field 1'),
```
Comment (EnTeQuAk) [inline src/olympia/discovery/data.py]: Hmm, the copy is kinda unclear on this. On the one hand it calls this just `Yellowfield` and in the ordering at the bottom it says `Yellow field 1` - I'm guessing the first one is the "correct" one.

Comment (diox) [inline src/olympia/discovery/data.py]: Yeah, good point, I'll fix that

**Comment by EnTeQuAk** [comment]
r+

**Pull request merged**